x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/slm/SnapshotInvocationRecord.java

```diff
@@ -29,19 +31,23 @@
     implements Writeable, ToXContentObject, Diffable<SnapshotInvocationRecord> {
 
     static final ParseField SNAPSHOT_NAME = new ParseField("snapshot_name");
+    static final ParseField START_TIMESTAMP = new ParseField("snapshot_start_time")
+        .forRestApiVersion(RestApiVersion.onOrAfter(RestApiVersion.V_8));
```

Comment:
Since this is an additive change, we don't need to add the rest compatibility stuff for this field

Reply:
I took this out (and made sure we could handle a null value for the field)